src/main/java/com/parasoft/demoapp/repository/industry/OrderRepository.java

```diff
@@ -16,7 +17,8 @@ public interface OrderRepository extends JpaRepository<OrderEntity, Long> {
 
     Page<OrderEntity> findAllByRequestedBy(String requestedBy, Pageable pageable);
 
-    int countByReviewedByPRCH(boolean b);
+    @Query("select count(reviewedByPRCH) FROM OrderEntity where reviewedByPRCH=false and requestedBy=:requestedBy")
+    int countByReviewedByPRCH(@Param("requestedBy") String requestedBy);
```

Comment:
We should use `int countByResuestByAndReviewedByPRCH(String requestedBy, boolean b);` instead of `@Query` annotation.